packages/website/src/examples/layout/Table/TableMultiSelection.demo.tsx

```diff
@@ -9,38 +18,45 @@ interface IExampleRowData {
     body: string;
 }
 
+const columnHelper = createColumnHelper<IExampleRowData>();
+const columns: Array<ColumnDef<IExampleRowData>> = [
+    columnHelper.accessor('userId', {
+        header: 'User ID',
+        cell: (info) => info.row.original.userId,
+        enableSorting: false,
+    }),
+    columnHelper.accessor('title', {
+        header: 'Title',
+        cell: (info) => info.row.original.title,
+        enableSorting: false,
+    }),
+];
+
 export default () => {
-    const columnHelper = createColumnHelper<IExampleRowData>();
-    const columns: Array<ColumnDef<IExampleRowData>> = [
-        columnHelper.accessor('userId', {
-            header: 'User ID',
-            cell: (info) => info.row.original.userId,
-        }),
-        columnHelper.accessor('title', {
-            header: 'Title',
-            cell: (info) => info.row.original.title,
-        }),
-    ];
-    const [data, setData] = useState([]);
-    const [loading, setLoading] = useState(false);
+    const [data, setData] = useState(null);
+    const [loading, setLoading] = useState(true);
     const [pages, setPages] = useState(1);
 
-    const fetchData = (state: any) => {
+    const fetchData: onTableChangeEvent = async (state) => {
         setLoading(true);
         const searchParams = new URLSearchParams({
-            _page: state.pagination.pageIndex + 1,
-            _limit: state.pagination.pageSize,
+            _page: '' + state.pagination.pageIndex + 1,
+            _limit: '' + state.pagination.pageSize,
```

Comment:
I would prefer if we called toString directly
```suggestion
            _page: (state.pagination.pageIndex + 1).toString(),
            _limit: state.pagination.pageSize.toString(),
```